packages/core/src/Utils/Hotjar/index.ts

```diff
@@ -1,11 +1,11 @@
 import { epochToMoment, toMoment } from '@deriv/shared';
 import { TCoreStores } from '@deriv/stores/types';
 
-const isProduction = process.env.NODE_ENV === 'production';
+const isProductionOrStaging = process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'staging';
 
 const initHotjar = (client: TCoreStores['client']) => {
     // To initialize only on licensed domains.
```

Comment:
Do you think it can be good idea to explicitly tell staging and production on the comment rather then licensed domains.